modules/gsqlite3backend/4.3.1_to_4.7.0_schema.sqlite3.sql

```diff
@@ -10,9 +10,9 @@ BEGIN TRANSACTION;
     last_check            INTEGER DEFAULT NULL,
     type                  VARCHAR(8) NOT NULL,
     notified_serial       INTEGER DEFAULT NULL,
+    account               VARCHAR(40) DEFAULT NULL
```

Comment:
```suggestion
    account               VARCHAR(40) DEFAULT NULL,
```

Reply:
oops, thanks! I did fix that in the normal schema file already. I think I'll add tests.